src/vec_buffers.h

```diff
@@ -475,6 +908,83 @@ void Buffers<T>::swap(unsigned i, unsigned j)
 {
     using std::swap;
     swap(mem[i], mem[j]);
+    if (m_meta) {
+        swap(meta[i], meta[j]);
+    }
+}
+
+/** Perform a preparatiion in radix-2 FFT algorithm
+ *
+ * @param input - input buffers
+ * @param scrambler - a vector stores bit-reversed values
+ * @param data_len - a conven length
+ * @param group_len - number of elements in which a same operations will be
+ * performed
+ */
+template <typename T>
+inline void Buffers<T>::radix2_fft_prepare(
```

Comment:
https://github.com/torvalds/linux/blob/master/Documentation/process/coding-style.rst#15-the-inline-disease :smiling_imp:

Reply:
It's updated. Thanks @NicolasT